ts/src/huobi.ts

```diff
@@ -3017,35 +3015,29 @@ export default class huobi extends Exchange {
         const isolated = (marginMode === 'isolated');
         const cross = (marginMode === 'cross');
         const margin = (type === 'margin') || (spot && (cross || isolated));
+        let response = undefined;
```

Comment:
hum found this crash:
```
➜  ccxt git:(update-huobi-apis-20231018) ✗ p huobi fetchBalance --swap > out.txt
Traceback (most recent call last):
  File "/Users/cjg/Git/ccxt12/ccxt/examples/py/cli.py", line 239, in <module>
    asyncio.run(main())
  File "/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/cjg/Git/ccxt12/ccxt/examples/py/cli.py", line 216, in main
    result = await method(*args)
             ^^^^^^^^^^^^^^^^^^^
  File "/Users/cjg/Git/ccxt12/ccxt/python/ccxt/async_support/huobi.py", line 3078, in fetch_balance
    for i in range(0, len(data)):
                      ^^^^^^^^^
TypeError: object of type 'NoneType' has no len()
```

Reply:
@carlosmiei can't reproduce this, how do you get this?
`p huobi fetchBalance '{"subType":"linear","unified":true,"margin":"cross"}' --swap`
`p huobi fetchBalance '{"subType":"linear","unified":false,"margin":"cross"}' --swap`